src/action/common/place_nix_configuration.rs

```diff
@@ -143,15 +151,6 @@ impl PlaceNixConfiguration {
             "(nix:$name)\\040".to_string(),
         );
         settings.insert("max-jobs".to_string(), "auto".to_string());
-        if let Some(ssl_cert_file) = ssl_cert_file {
-            let ssl_cert_file_canonical = ssl_cert_file.canonicalize().map_err(|e| {
-                Self::error(ActionErrorKind::Canonicalize(ssl_cert_file.to_owned(), e))
-            })?;
-            settings.insert(
-                "ssl-cert-file".to_string(),
-                ssl_cert_file_canonical.display().to_string(),
-            );
-        }
```

Comment:
are we worried about users setting this in, now nix.custom.conf, removing it, and breaking things? bit curious about this part of these changes.

Reply:
I'm not sure I follow. I moved it because it isn't part of our standard config, but instead user-specified.

Reply:
(The question was really "why did this move" and the answer is "because it's definitely user-specified config")